cmd/sonobuoy/app/status.go

```diff
@@ -80,10 +80,21 @@ func getStatus(cmd *cobra.Command, args []string) {
 	} else {
 		err = printSummary(os.Stdout, status)
 	}
+
 	if err != nil {
 		errlog.LogError(err)
 		os.Exit(1)
 	}
+	os.Exit(exitCode(status))
+}
+
+func exitCode(status *aggregation.Status) int {
+	switch status.Status {
+	case aggregation.FailedStatus:
```

Comment:
Why is this not an error on GetStatus?

Reply:
because it's not an error getting the status -- the error is on the plugin and is reported through the status.

Reply:
I don't have an example, but it will look like this (hand crafted output):

```
test salazar:sonobuoy cha$ sonobuoy status
PLUGIN		STATUS		COUNT
e2e		complete	1
systemd_logs	failed	1

Sonobuoy has failed. You can see what happened with `sonobuoy logs`.

test salazar:sonobuoy cha$ echo $?
1
```